docs/installation.md

```diff
@@ -2,73 +2,102 @@
 title: Installation
```

Comment:
I see this section as "Native installation" and the one in the Melange section as "Js installation". Not sure if it'd be worth to separate these 2 cases more clearly (prob to do somewhere in the future).

Reply:
The biggest difference is just melange right?

We might need to revisit this in the future, indeed.